### PR TITLE
PostgreSQL - closing connection after executing the statement

### PR DESCRIPTION
**Pull request by gshriki**

### CODE REVIEW

File: Integrations/integration-PostgreSQL.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ script:
             table.append(tableRow)
 
         demisto.results( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': contents, "ReadableContentsFormat": formats['table'], "HumanReadable": table} );
-
+        conn.close()
```
Comment (dorsha) [inline Integrations/integration-PostgreSQL.yml]: You should also do cur.close() before

image

**Comment by dorsha** [comment]
You should also close the connection in the Test mode........ 

image

**Comment by dorsha** [comment]
and not only in the command...
Please create a function for closing the connection, so everyone will call it if needed, it will be better for maintenance later.

**Comment by slavikm** [comment]
As usual, for languages that support it, the right way to do this is using `finally`.

**Comment by dorsha** [comment]
Beside the above comments, another issue I found with this fix is that if you have **syntax** error, the connection was not closed as well..

Anyhow, I fixed all of my code review comments and issues.

**Pull request merged**